src/reference/asciidoc/whats-new.adoc

```diff
@@ -8,3 +8,9 @@ This version requires the 1.0.0 `kafka-clients` or higher.
 
 The `StringJsonMessageConverter` and `JsonSerializer` now add type information in `Headers`, allowing the converter and `JsonDeserializer` to create specific types on reception, based on the message itself rather than a fixed configured type.
 See <<serdes>> for more information.
+
+
+==== Container Stopping Error Handlers
+
+Container Error handlers are now provided for both record and batch listeners that treat any exceptions thrown by the listener as fatal; they stop the container.
```

Comment:
> ... and rethrow a `KafkaException` to prevent further processing.

Or something similar.

Reply:
That's not really appropriate (too much detail) for a "what's new" entry. However, it raised an issue in my mind about the seek to current error handlers.

Polishing now...

Reply:
Oh! Really?
So, tell that in the main Doc then, please.

I think if we had it before I might not ask the question why we re-throw...

Thanks